as/src/base/service.c

```diff
@@ -760,9 +760,8 @@ stop_service(thread_ctx* ctx)
 	cf_poll_destroy(ctx->poll);
 	cf_epoll_queue_destroy(&ctx->trans_q);
 
-	cf_free(ctx);
-
 	cf_detail(AS_SERVICE, "stopped ctx %p", ctx);
+	cf_free(ctx);
```

Comment:
Did GCC warn about this?

`ctx` isn't being de-referenced here, we are only logging the value of the pointer.